src/shared/locales/index.js

```diff
@@ -34,6 +34,10 @@ export const availableLanguages = [
     value: 'pt-BR',
     label: 'Português (Brasil)'
   },
+  {
+    value: 'ru-RU',
```

Comment:
I think this locale key is wrong, you should use `ru` instead of `ru-RU`.

https://electronjs.org/docs/api/locales

> ru | Russian